src/applications/disability-benefits/526EZ/actions/index.js

```diff
@@ -1,64 +1,132 @@
-/* eslint-disable no-unused-vars */
 import { apiRequest } from '../../../../platform/utilities/api';
-import { fetchInProgressForm } from '../../../common/schemaform/save-in-progress/actions';
+import Raven from 'raven-js';
 
-export const ITFStatuses = Object.freeze({
+import { getSortedTimestamps } from '../helpers';
+
+export const PRESTART_STATUS_SET = 'PRESTART_STATUS_SET';
+export const PRESTART_STATUS_RESET = 'PRESTART_STATUS_RESET';
+export const PRESTART_DISPLAY_RESET = 'PRESTART_DISPLAY_RESET';
+
+export const PRESTART_STATUSES = {
+  notAttempted: 'not-attempted',
   active: 'active',
-  claim_received: 'claim_received', // eslint-disable-line camelcase
-  duplicate: 'duplicate',
-  expired: 'expired',
   pending: 'pending',
-  incomplete: 'incomplete'
-});
+  none: 'none',
+  expired: 'expired',
+  created: 'created',
+  retrieved: 'retrieved',
+  renewed: 'renewed',
+  notRetrievedSaved: 'not-retrieved-saved',
+  notRetrievedNew: 'not-retrieved-new',
+  notCreated: 'not-created',
+  notRenewed: 'not-renewed',
+};
```

Comment:
Swagger for `/v0/intent_to_file` shows five statuses possible for each ITF entry in the array:
`[active, claim_recieved, duplicate, expired, incomplete]`

Whereas `PRESTART_STATUSES` seems to like a combination of ITF statuses (`active`, `expired`) and possible states for a veteran's overall ITF status (`notRetrievedSaved`, etc.).

Would it make sense to break these up for clarity?

Reply:
Interesting idea, so it looks like we'll stick closer with the original configuration and keep these in `status` (since these reflect application state), and then in `data` store the following to reflect what's returned from the endpoint: `currentStatus`, `currentExpirationDate`, and if we renew the ITF, also store `previousExpirationDate`. How does that sounds @cvalarida @markgreenburg ?

Reply:
Also instances where there are no `expired` ITF's should also be included in the `none` state, will update.

Reply:
Actually we never use the ITF status itself outside of the response handler, so we can simply store `currentExpirationDate` and `previousExpirationDate` in `data`.

Reply:
I'm not sure I understand 100%, but I'm interested to see where it goes. It might be easier to code than to explain (as much as I hate to say it since if we can't explain our code clearly, we're probably complicating things).

Reply:
@cvalarida, how about splitting status up into status and message?
status types: not_attempted, pending, succeeded, failed
message types: retrieved, created, etc.

Reply:
Seems like messages should be derived from the state of the application as a whole, so I think `message_type` is perhaps to prescriptive? Like we're trying to define behavior, not state, but behavior should be derived _from_ state, y'know?

Reply:
Interesting, I had considered this as a type of [UI state](https://tonyhb.gitbooks.io/redux-without-profanity/content/ui_state.html), since it's primarily used to determine which message is displayed without having any other implications for the form. But I haven't thought about it more deeply than that, so if it makes sense to calculate these later than we can do that instead.